test/e2e/gateway/tokens_only.go

```diff
@@ -0,0 +1,144 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gateway
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/hyperledger/firefly-common/pkg/fftypes"
+	"github.com/hyperledger/firefly/pkg/core"
+	"github.com/hyperledger/firefly/test/e2e"
+	"github.com/hyperledger/firefly/test/e2e/client"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type TokensOnlyTestSuite struct {
+	suite.Suite
+	testState   *testState
+	connector   string
+	db          string
+	key         string
+	adminHost1  string
+	configFile1 string
+}
+
+func (suite *TokensOnlyTestSuite) SetupSuite() {
+	suite.testState = beforeE2ETest(suite.T())
+	stack := e2e.ReadStack(suite.T())
+	stackState := e2e.ReadStackState(suite.T())
+
+	adminProtocol1 := schemeHTTP
+	if stack.Members[0].UseHTTPS {
+		adminProtocol1 = schemeHTTPS
+	}
+	suite.adminHost1 = fmt.Sprintf("%s://%s:%d", adminProtocol1, stack.Members[0].FireflyHostname, stack.Members[0].ExposedAdminPort)
```

Comment:
A lot of this stuff is now in `testState` - see https://github.com/hyperledger/firefly/commit/6a2fc3cff8ac31d3810c73e28bbd0eb0d75037dc

Reply:
I think it's only in `multiparty` `beforeE2E()`, but I can do the same here